### ISSUE TITLE
Change the label from "Edit this page" to "Improve this page"

### ISSUE THREAD

**Issue by alexearnshaw** [Axway/axway-open-docs#86]
- Update the "Edit this page" button to "Improve this page". 
- Update the "Edit on Netlify CMS" to match.
- Update any docs (on netlify or confluence etc) that use the old button names
- Review the links in this sidebar to check if they are absolute or relative and what they should be?

**Comment by alexearnshaw** [comment]
After reviewing the change to "Improve" we decided it was not clear or meaningful enough, especially when both links were changed to "Improve", so we decided on "Edit on GitHub" and "Edit on Netlify CMS".

On the links themselves it is not possible to link to the deploy preview unless we use a different base URL to build hugo on the branch so these links to CMS/Contrib guidelines will continue to point to the main site for now.

**Comment by andreamussap** [comment]
RE Improve vs. Edit -> We searched how other open source projects are doing and we found both options. The major difference between our project and theirs is that they only have one way to edit, that is, on GitHub, so it's easier to have a button 'Improve this page', but in our case, it's important to specify "how" your're going to improve the page: on Git or on Netlify, and using 'Improve this page on GitHub' and 'Improve this page on Netlifly CMS' is long and unnecessary. I'm happy with the new labels.
